sdk/servicebus/azure-servicebus/tests/perf_tests/_test_base.py

```diff
@@ -5,124 +5,279 @@
 
 import uuid
 
-from azure_devtools.perfstress_tests import PerfStressTest, get_random_bytes
+from azure_devtools.perfstress_tests import PerfStressTest, get_random_bytes, BatchPerfTest
 
-from azure.servicebus import ServiceBusClient, ServiceBusReceiveMode, ServiceBusMessage
+from azure.servicebus import ServiceBusClient, ServiceBusReceiveMode, ServiceBusMessage, TransportType
 from azure.servicebus.aio import ServiceBusClient as AsyncServiceBusClient
 from azure.servicebus.aio.management import ServiceBusAdministrationClient
 
-MAX_QUEUE_SIZE = 40960
-
-
-class _ServiceTest(PerfStressTest):
-    service_client = None
-    async_service_client = None
-
-    def __init__(self, arguments):
+class _ReceiveTest():
+    def __init__(self, arguments) -> None:
         super().__init__(arguments)
```

Comment:
We're calling super, but there's no super?

Reply:
If this is intended to be a mixin - I would probably remove the constructor and move this code to some kind of setup method.
Having constructors with multiple inheritance can be hard to follow.